test/system/206-user.bats

```diff
@@ -512,9 +518,241 @@ teardown() {
   run --keep-empty-lines --separate-stderr "$TOOLBX" run --distro ubuntu --release 20.04 cat /etc/group
 
   assert_success
+  assert_line --regexp "^$USER:x:[[:digit:]]+:$USER$"
   assert_line --regexp "^sudo:x:[[:digit:]]+:$USER$"
   assert [ ${#lines[@]} -gt 1 ]
 
   # shellcheck disable=SC2154
   assert [ ${#stderr_lines[@]} -eq 0 ]
 }
+
+@test "user: id(1) for $USER inside the default container" {
```

Comment:
These tests fail if I revert the crun [fix](https://github.com/containers/crun/commit/9effaebb429a1aed).  So, I suppose they serve their purpose.